### PR TITLE
Fixes and Additions to New DST Food/Recipes; Correction on Caviar and Feijoada Recipes

### PR DESCRIPTION
**Pull request by lakhnishMonster**
1. Added ingredient values (& images) for Volt Goat Horns, Nightmare Fuel, and Bone Shards.

2. Updated Lesser Glowberry; its allowed in crockpot now as a fruit.

3. Fixed recipes for Fancy Spiralled Tubers & Creamy Potato Purée.
- 	The Fancy Spiralled Tubers was missing twigs, which was required.
- 	The Creamy Potato Purée was changed to follow the listed logic in preparedfoods.lua for DST.

4. Moved Juicy Berries up the list to follow release date instead of Warly's section.

5. Removed the decoration value from Lune Tree Blossom (doesn't have such a thing).

6. Reverted recipes that ignored issue #32; Butter Muffin, Waffles, Trail Mix, & Guacamole.
- The DST versions of those recipes have been commented out. It can be used for when there is support for it.

7. Reverted the incorrect change made to the Caviar recipe: 
- The change made to caviar was to make it check for a veggie value > 1. There is no such check in preparedfoods.lua. A 0.5 veggie unit (like a mushroom) + 3 cooked roe will indeed yield caviar (I tested to make sure as well).

8. Reverted the correct-yet-incorrect change made to the Feijoada recipe:
-  The reverted version was following what was listed in preparedfoods.lua. Issue #28 is correct that the recipe itself is bugged, but it's bugged in preparedfoods.lua, not the simulator (which was following what was listed in said lua file). Klei would need to fix that first on their end before it makes it way to the simulator (good luck fixing that xD). 
- For now, the recipe follows Klei's version, but is listed as a note so that people can see this behavior when using the simulator.

### CODE REVIEW

**Comment by lakhnishMonster** [comment]
With this, the only things left are foods that exist across versions and Warly's foods for DST, both of which needs addressing.

**Comment by 6lancmange** [comment]
It would be great if you could leave the DST-corrected recipes for Waffles, Butter Muffin, Trail Mix, and Guacamole commented out, so it's easier to re-add them when the whole issue is worked out.

I find it's better to have the recipes allowing both possible ingredients even if you don't select DST than not have them when you're trying to find recipes for DST. I'm gonna keep using the local copy for now anyway. Thanks for correcting the potato recipes to match the game file!

File: html/foodguide.js [hunk 1]
```diff
@@ -3055,9 +3070,9 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 			mashedpotatoes: {
 				name: 'Creamy Potato Purée',
 				test: function(cooker, names, tags) {
-					return ((names.potato >= 2 || names.potato_cooked >= 2) || ((names.potato + names.potato_cooked) >= 2)) && (names.garlic || names.garlic_cooked) && !tags.meat && !tags.inedible;
+					return ((names.potato && names.potato > 1) || (names.potato_cooked && names.potato_cooked > 1) || (names.potato && names.potato_cooked)) && (names.garlic || names.garlic_cooked) && !tags.meat && !tags.inedible,
```
Comment (6lancmange) [inline html/foodguide.js]: I think there should be a semicolon at the end. Same goes for potatotornado.

**Comment by 6lancmange** [comment]
The image names are wrong. They should be the same as the ingredient names in game, not the code ones, just all lowercase and with _ instead of spaces.

**Pull request merged**